prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -143,6 +165,42 @@ c2s_interfaces = { "*", "::" }
 c2s_interfaces = { "*" }
 {{ end }}
 
+{{ if $ENABLE_S2S -}}
+-- set s2s port
+s2s_ports = { {{ $S2S_PORT }} } -- Listen on specific s2s port
+
+s2s_whitelist = {
+	{{ if $ENABLE_VISITORS -}}
+    'conference.{{ $XMPP_DOMAIN }}'; -- needed for visitors to send messages to main room
```

Comment:
This should probably use the variable or won't work, will it?

Reply:
Excellent point, good catch.